stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -393,11 +418,12 @@ function externEncryptAesCbc(byte[] input, byte[] key, byte[] iv, handle padding
 } external;
 
 # Returns AES ECB encrypted value for the given data.
+# ```ballerina byte[]|crypto:Error cipherText = crypto:encryptAesEcb(cipherText, key); ```
 #
 # + input - The content to be encrypted
 # + key - Encryption key
 # + padding - The padding
-# + return - Encrypted data or `Error` if key is invalid
+# + return - Encrypted data or `crypto:Error` if key is invalid
```

Comment:
```suggestion
# + return - Encrypted data or `crypto:Error` if key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.